git_pull/run.sh

```diff
@@ -41,17 +39,17 @@ function add-ssh-key {
 function git-clone {
     # create backup
     BACKUP_LOCATION="/tmp/config-$(date +%Y-%m-%d_%H-%M-%S)"
-    echo "[Info] Backup configuration to $BACKUP_LOCATION"
+    bashio::log:info "Backup configuration to $BACKUP_LOCATION"
 
-    mkdir "${BACKUP_LOCATION}" || { echo "[Error] Creation of backup directory failed"; exit 1; }
-    cp -rf /config/* "${BACKUP_LOCATION}" || { echo "[Error] Copy files to backup directory failed"; exit 1; }
+    mkdir "${BACKUP_LOCATION}" || { bashio::log:error "Creation of backup directory failed"; exit 1; }
```

Comment:
You can do it like this:
```suggestion
    mkdir "${BACKUP_LOCATION}" || bashio::exit.nok "Creation of backup directory failed"
```

Reply:
This applies to all occurrences of course...